opflexagent/opflex_notify.py

```diff
@@ -177,7 +177,9 @@ def __init__(self):
 
         def run(self):
             self.agent = OpflexNotifyAgent()
-            self.agent.run()
+            # Don't bother running if we don't have a socket
+            if self.agent.sockname:
+                self.agent.run()
```

Comment:
maybe the run method should be a noop if there's no sockname (logging something meanwhile) instead of leaving the check to the caller.

Reply:
Sounds good -- will fix.